.github/workflows/post-merge.yml

```diff
@@ -743,6 +734,26 @@ jobs:
         uses: github/codeql-action/upload-sarif@ce28f5bb42b7a9f2c824e633a3f6ee835bab6858  # v3.29.0
         with:
           sarif_file: trivy-${{ env.sanitized_image_name }}.sarif
+  version-bump:
```

Comment:
we have a `run_version_dev` in the parameters, should we guard it with that?

Reply:
yes. good point. I'll use that one